src/pages/profile-page/ui/profile-page.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useStore } from "effector-react";
+
+import { $auth, $user } from "entities/session";
```

Comment:
in index.ts "entities/session" make only one export "export * as sessionModel from './model';"
And on page import sessionModel  ande use it - sessionModel.$auth

Reply:
got it. changed